docs/source/example/dscp-app.md

```diff
@@ -0,0 +1,45 @@
+
+# DSCP - Digital Supply Chain Platform
+
+### A sample supply chain application deployed on Substrate based DLT network using Hyperledger Bevel <br>
+<br>
+This demo application has been developed to demonstrate the benefits of distributed systems to the additive-manufacturing industry within aerospace. The application defines a consortium of multiple organizations. It allows nodes to track products or goods along with their chain of custody. It provides the members of the consortium all the relevant data about their product throughout it's journey, from raw material to the finished product. It provides simple web clients that let you perform the actions of different personas in a supplychain industry.<br>
+<br>
+
+### Pre-requisites
+
+***
+
+- Substrate network of one or more organizations. A complete supplychain network will have the following organizations:
+  - Supplychain (admin organization)
+  - Carrier
+  - Store
+  - Warehouse
+  - Manufacturer
+- IPFS node
+
+### Setup Guide
+
+***
+
+DSCP-API
+
+This application uses a node.js API to support communication to the Substrate based dscp-node and an IPFS node via polkadot.js API.
+Follow the steps in this [guide](https://github.com/inteli-poc/bevel/tree/develop/examples/dscp-app) for dscp-api configuration.
```

Comment:
Its always better to use relative links for documentation instead of absolute.

e.g `[guide](examples/dscp-app)`

Reply:
This is for our 'read the docs' site and require absolute links to be clickable.

Reply:
ah the curse of sphinx RST